src/components/auth/LoginForm.jsx

```diff
@@ -28,11 +27,8 @@ const LoginForm = ({ onSubmit, formErrors, formData, handleChange }) => {
             >
                 Password
             </FormInput>
-            {/* <span className="flex items-center gap-1 mb-2">
-                <input id="remember" type='checkbox' className="h-4 w-4 accent-red" />
-                <label htmlFor="remember" className="font-spartan">Remember me</label>
-            </span> */}
-            <button type="submit" className="bg-red w-full hover:bg-pureWhite hover:text-red hover:border-2 hover:border-red text-white font-spartan font-semibold text-lg py-1 px-7 rounded-lg">Log In</button>
+            <button type="button" onClick={onForgotPassword} className="text-lightGreen mt-2 mb-2">Forgot Password?</button>
```

Comment:
add to the classNames font-spartan and change text-lightGreen to text-red 
I think it would look better

Reply:
When designing a "Forgot Password" link, blue/green/black is generally the preferred color. This is because blue is commonly associated with links and is perceived as trustworthy, calming, and easy on the eyes. Red, on the other hand, is often associated with warnings or errors and might give a sense of urgency or caution, which isn't ideal for a "Forgot Password" link. And we have red button Login, so it could be too much red.

Reply:
It is am interesting question. You do not have the mock-up for this particular page, but you already has idea how the pages should look like. And the app has the button color. What do you think if you'll discuss design question with team when implementing something new?